packages/angular/projects/clr-angular/src/data/datagrid/datagrid.ts

```diff
@@ -158,6 +158,8 @@ export class ClrDatagrid<T = any> implements AfterContentInit, AfterViewInit, On
   @Input() clrDgSingleSelectionAriaLabel: string = this.commonStrings.keys.singleSelectionAriaLabel;
   @Input() clrDgSingleActionableAriaLabel: string = this.commonStrings.keys.singleActionableAriaLabel;
   @Input() clrDetailExpandableAriaLabel: string = this.commonStrings.keys.detailExpandableAriaLabel;
+  // Allows disabling of the auto focus on page/state changes (excludes focus management inside of popups)
+  @Input() clrDgDisableAutoFocus = false;
```

Comment:
Can't we specifically say `clrDgDisablePageFocus`?

Reply:
I'll change it, thanks for the suggestion.